src/components/configuration/Configuration.ts

```diff
@@ -96,7 +96,7 @@ export class Configuration {
             undefined
         );
 
-        configuration.updatePathToRlsExecutableSpecifiedByUser();
+        configuration.updateUserRlsPath();
```

Comment:
I find it more difficult to understand what "User RLS Path" is.
This is why my method has verbose name.
If you really want to rename it, I would suggest `updateRlsPathSpecifiedByUser`.